### DISCUSSION TITLE
Does CLO mean anything in particular ?

### DISCUSSION THREAD

**Discussion by pgonin** [cncf/clomonitor#746] [Q&A] [answered]
I was looking in multiple places but I could not find if CLO was an acronym. Missing the obvious?

**Comment by caniszczyk** [comment, marked as answer]
CLO is the first three letters of "Cloud Native Computing Foundation"

We were essentially building a monitoring tool and wanted a unique name, so "CNCF" + "monitor" = CLOMonitor :)
